quiz.rb

```diff
@@ -2,19 +2,60 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+
+  attr_reader :name
```

Comment:
Nice job! Pro tip : rather than putting these on separate lines, you can actually put them all on one line, and separate the symbols with commas, e.g.
`attr_reader :name, :breed, :age`